docs/source/model_doc/wav2vec2.rst

```diff
@@ -143,3 +136,19 @@ FlaxWav2Vec2ForPreTraining
 
 .. autoclass:: transformers.FlaxWav2Vec2ForPreTraining
     :members: __call__
+
+Wav2Vec2 specific outputs
```

Comment:
In all other doc pages, those go before the models, so let's leave them here for now. We can decide to switch them after the model but all models together?

Reply:
Sounds good - reverting this